packages/components/tabs/src/tabs.js

```diff
@@ -0,0 +1,84 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './tabs.css';
+import { customEventNames } from './utils';
+
+customElementDefineHelper(
+	'ts-tabs',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				// tab properties: {label, icon, selected, closeable, counter}
+				tabs: { type: Array, reflect: true },
+				dir: { type: String, reflect: true }
+			};
+		}
+
+		get direction() {
+			return this.dir ? this.dir : this.bodyDir;
+		}
+
+		tabClickHandler(tab) {
+			console.log('tab', tab);
+			this.dispatchCustomEvent(customEventNames.TAB_CLICK, { tab });
+		}
+
+		iconTemplate(tab) {
+			if (!tab.icon) {
+				return;
+			}
+			return html`
+				<ts-icon icon="${tab.icon}" size="medium"></ts-icon>
+			`;
+		}
+
+		badgeTemplate(tab) {
+			if (!tab.counter) {
+				return;
+			}
+			return html`
+				<em class="badge">${tab.counter}</em>
+			`;
+		}
+
+		closeButtonTemplate(tab) {
+			if (!tab.closeable || !tab.selected) {
+				return;
+			}
+			return html`
+				<ts-icon class="close-button" icon="close" size="large"></ts-icon>
+			`;
+		}
+
+		tabTemplate(tab) {
+			/* eslint-disable */
```

Comment:
Good catch! I forgot to mention the reason! 

The issue is for the listener that I want to pass the `tab` data to the event handler

Reply:
57:51  error  Arrow functions and `.bind` must not be used in templates, a method should be passed directly like `${this.myMethod}` as it will be bound automatically  lit/no-template-bind

Reply:
Maybe there is another way around this that I couldn't think of to pass the tab data
In general it's a good rule but need to be disabled purposefully in my opinion.